tests/plugins/clipboard/manual/pasteimagehtml.html

```diff
@@ -0,0 +1,7 @@
+<div id="editor">
```

Comment:
Shouldn't this test be ignored on all browsers but Firefox?

Reply:
I didn't ignore it as we will probably use the code for Firefox also in other browsers to handle pasting images (see https://github.com/ckeditor/ckeditor4/issues/2675#issuecomment-515750101). Therefore not ignoring the test is more future-proof.